dataviz/test/nuxeo-audit-data-test.html

```diff
@@ -212,6 +213,7 @@
               min: '1980-01-01',
               max: '1980-01-03',
             },
+            time_zone: moment().format('Z'),
```

Comment:
If we are not testing it, why do we need to specify a value here, since `moment().format('Z')` is already the default value?

Reply:
Not sure I got it. Elasticsearch backend needs to know in which timezone it has to serialize bucket keys...

Reply:
Nvm, I should have expanded the code above this line.